ktor-client/ktor-client-core/common/src/io/ktor/client/request/forms/FormDataContent.kt

```diff
@@ -31,14 +31,18 @@ class FormDataContent(
     override fun bytes(): ByteArray = content
 }
 
+private val buildFormDataContentType = { boundary: String ->
+    ContentType.MultiPart.FormData.withParameter("boundary", boundary)
+}
 
 /**
  * [OutgoingContent] for multipart/form-data formatted request.
  *
  * @param parts: form part data
  */
 class MultiPartFormDataContent(
-    parts: List<PartData>
+    parts: List<PartData>,
+    buildContentType: (boundary: String) -> ContentType = buildFormDataContentType
```

Comment:
Could we move `boundary` and `contentType` fields in the constructor?

Reply:
@e5l sure! pushed the fix!